internal/command/command.go

```diff
@@ -133,7 +133,10 @@ var CmdConfigure = &Command{
 		if err := container.Generate(ctx, image, apiRoot, outputDir, generatorInput, flagAPIPath); err != nil {
 			return err
 		}
-		// No need to clean here, as configure should fail for an existing API.
+		// We don't need to clean the newly-configured API, but we *do* need to clean any non-API-specific files.
+		if err := container.Clean(ctx, image, languageRepo.Dir, "none"); err != nil {
```

Comment:
What's the difference between `clean --api-path=none` and not having an `--api-path` argument?

Reply:
No --api-path means "clean all configured APIs" at the moment. I'm tempted to make it required and have two special values: `all` and `none` for clarity. But that's probably a 2025 job.